File: 03_conditions/python/login_check.py
# login_check.py
# Python: and を使った条件分岐

username = input("ユーザー名: ")
password = input("パスワード: ")

if username == "admin" and password == "python":
    print("ログイン成功")
else:
    print("ログイン失敗")

File: 03_conditions/go/login_check.go
package main

import "fmt"

func main() {
	username := "student"
	password := "linux"

	if username == "student" && password == "linux" {
		fmt.Println("ログイン成功")
	} else {
		fmt.Println("ログイン失敗")
	}
}
